kernel/src/main/java/org/kframework/kompile/KompileOptions.java

```diff
@@ -132,4 +132,7 @@ public boolean isKore() {
 
     @Parameter(names="--top-cell", description="Choose the top configuration cell when more than one is provided. Does nothing if only one top cell exists.")
     public String topCell;
+
+    @Parameter(names={"--post-process"}, description="JSON KAST => JSON KAST converter to run on definition after kompile pipeline.")
+    public String postProcess;
```

Comment:
KAST is a lossy format meant for readability.
To make sure we can output the definition and load it back we need to use KORE.
The JSON format is not ready right now.
This PR should try and fix one of the issues: https://github.com/runtimeverification/k/pull/2572
But there might be more inconsistencies.
So I would mark this PR as blocked for now.

Reply:
Done, I've marked it as blocked.